examples/pico-examples/hello.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2021 Jeff Epler, written for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+#
+# Adapted from the example https://github.com/raspberrypi/pico-examples/tree/master/pio/hello_pio
+
+import time
+import board
+import rp2pio
+import adafruit_pioasm
+
+hello = """
+.program hello
+loop:
+    pull
+    out pins, 1
+    jmp loop
+"""
+
+assembled = adafruit_pioasm.assemble(hello)
+
+sm = rp2pio.StateMachine(
+    assembled,
+    frequency=80,
```

Comment:
The state machine can't go this slow and CP should now raise an error to tell you that.